### PR TITLE
Update message ID

### PR DESCRIPTION
**Pull request by pmd1nh**

### CODE REVIEW

**Comment by pmd1nh** [comment]
relate to https://github.com/OpenLiberty/open-liberty/pull/27494

Add message ID

#build        #spawn.fullfat.buckets=io.openliberty.webcontainer.servlet.6.1.internal_fat,io.openliberty.webcontainer.servlet.6.0.internal_fat,com.ibm.ws.webcontainer.servlet.6.0_fat_toleration.lWAS,com.ibm.ws.webcontainer_fat,com.ibm.ws.webcontainer.servlet.3.1_fat,com.ibm.ws.webcontainer.servlet.4.0_fat,com.ibm.ws.webcontainer_fat_part2,com.ibm.ws.webcontainer_fat_servlet31toleration,com.ibm.ws.webcontainer_fat_servlet31toleration_part2,com.ibm.ws.webcontainer.servlet.4.0_fat_toleration.3.0,com.ibm.ws.webcontainer.servlet.4.0_fat_toleration.3.0_part2,com.ibm.ws.webcontainer_fat_lWAS,com.ibm.ws.webcontainer_fat_servlet31_webcontainerlWAS,com.ibm.ws.webcontainer.servlet.4.0_fat_toleration.3.0.lWAS

File: dev/com.ibm.ws.webcontainer/resources/com/ibm/ws/webcontainer/resources/Messages.nlsprops [hunk 1]
```diff
@@ -1632,4 +1632,6 @@ redirect.location.cannot.be.null.useraction=Update the redirect location by spec
 
 sendRedirect.hyper.text.url=<html><body><p>Redirecting to <a href="{0}"></a></p></body></html>
 
-query.or.post.is.null=Request query string or post data is null.
+query.or.post.is.null=CWWWC0009E: Request query string or post data is null.
```
Comment (dmuelle) [inline dev/com.ibm.ws.webcontainer/resources/com/ibm/ws/webcontainer/resources/Messages.nlsprops:1635]:
```suggestion
query.or.post.is.null=CWWWC0009E: The request query string or the post data is null.
```

File: dev/com.ibm.ws.webcontainer/resources/com/ibm/ws/webcontainer/resources/Messages.nlsprops [hunk 2]
```diff
@@ -1632,4 +1632,6 @@ redirect.location.cannot.be.null.useraction=Update the redirect location by spec
...
+query.or.post.is.null.explanation=Application tries to parse a null request parameter or a post body.
```
Comment (dmuelle) [inline dev/com.ibm.ws.webcontainer/resources/com/ibm/ws/webcontainer/resources/Messages.nlsprops:1636]:
```suggestion
query.or.post.is.null.explanation=The application tried to parse a null request parameter or an invalid post body.
```

File: dev/com.ibm.ws.webcontainer/resources/com/ibm/ws/webcontainer/resources/Messages.nlsprops [hunk 3]
```diff
@@ -1632,4 +1632,6 @@ redirect.location.cannot.be.null.useraction=Update the redirect location by spec
...
+query.or.post.is.null.useraction=Client needs to send a non-null request parameters or a valid post body.
```
Comment (dmuelle) [inline dev/com.ibm.ws.webcontainer/resources/com/ibm/ws/webcontainer/resources/Messages.nlsprops:1637]:
```suggestion
query.or.post.is.null.useraction=The client must send non-null request parameters and a valid post body.
```

**Review by dmuelle — CHANGES_REQUESTED** [review]
ID review

**Review by dmuelle — DISMISSED** [review]

**Comment by dmuelle** [comment]
dismissing my review as I was unaware @is273 was already in process of reviewing

File: dev/com.ibm.ws.webcontainer/resources/com/ibm/ws/webcontainer/resources/Messages.nlsprops (hunk 1 above)
Comment (is273) [inline dev/com.ibm.ws.webcontainer/resources/com/ibm/ws/webcontainer/resources/Messages.nlsprops]: Change "Request query string or post data is null." to "The HTTP request query string or the HTTP POST request data is null."

File: dev/com.ibm.ws.webcontainer/resources/com/ibm/ws/webcontainer/resources/Messages.nlsprops (hunk 2 above)
Comment (is273) [inline dev/com.ibm.ws.webcontainer/resources/com/ibm/ws/webcontainer/resources/Messages.nlsprops]: Change "Application tries to parse a null request parameter or a post body." to "The application tried to parse a null HTTP request parameter or a null HTTP POST request body."

File: dev/com.ibm.ws.webcontainer/resources/com/ibm/ws/webcontainer/resources/Messages.nlsprops (hunk 3 above)
Comment (is273) [inline dev/com.ibm.ws.webcontainer/resources/com/ibm/ws/webcontainer/resources/Messages.nlsprops]: Change "Client needs to send a non-null request parameters or a valid post body." to "The client must send a non-null HTTP request parameter or a non-null HTTP POST request body."

**Comment by pmd1nh** [comment]
#libby
#build

**Comment by is273** [comment]
@pmd1nh Approved the message

**Review by pnicolucci — APPROVED** [review]

**Comment by pmd1nh** [comment]
#libby

**Comment by pmd1nh** [comment]
```
no component 91341 1 0
 com.ibm.ws.app.manager_fat FAT tests 33 1 0
  componenttest.application.manager.test.SharedLibUpdateJarTest 1 1 0 4 s 7
   testSharedLibUpdateJar 1 1 0 4 s 1
```

**Pull request merged**